index.html

```diff
@@ -10189,7 +10191,7 @@ <h2>Screen capture</h2>
   then return <a>error</a> with <a>error code</a> <a>unable to capture screen</a>.
 
  <li><p>Let <var>file</var> be
-  <a>a serialization of the <code>canvas</code> element’s bitmap as a file</a>,
+  [=a serialization of the bitmap as a file|a serialization of the <code>canvas</code> element’s bitmap as a file=],
```

Comment:
Oh, I think this one with `<code>` inside the link text is broken.

Reply:
Good catch. Fixing this made me realize that this isn't exported from html anyway, so I restored the link in the index.